server/Dockerfile

```diff
@@ -9,5 +9,5 @@ COPY . /app
 RUN npm install
 
 RUN apt purge -y software-properties-common ca-certificates wget curl ssh git python build-essential && apt autoremove -y
-
+ENV OPENSSL_CONF=/dev/null
```

Comment:
why is that needed?

Reply:
As we tested it, there is some issue without that setting for the OpenSSL conf dir. I send you the error and from what we discussed, let's merge it like that for now.

Reply:
yes fine